src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs

```diff
@@ -40,9 +41,18 @@ module internal ProjectFile =
         |> Array.distinct
 
     let findReferencesFromResolutions (resolutions:Resolution array) =
+
+        let equals (s1:string, s2:string) =
```

Comment:
Why the tupled argument?

Reply:
I've been looking at C# code recently, sorry.

Reply:
You monster.

Reply:
I am indeed quite monstrous.  TBH, I actually like tupled funcs better than curried ones, I find it somehow ... cleaner.